frontend/components/Nav/Nav.jsx

```diff
@@ -0,0 +1,35 @@
+'use client';
+import Link from 'next/link';
+import { useLayoutEffect, useState } from 'react';
+import { usePathname } from 'next/navigation';
+import LogoutButton from '../LogoutButton';
+import navStyle from './Nav.module.css';
+
+const Nav = () => {
+    const router = usePathname();
+    const [ isSelected, setIsSelected ] = useState('');
+    
+    useLayoutEffect(() => {
+        setIsSelected(router);
+    }, [router])
+    return (
+        <nav className={navStyle.container}>
+            <ul>
+                <li className={isSelected === '/dashboard' ? navStyle.highlighted : undefined}>
```

Comment:
Repeated code here. If you create an array with all of this data, you can loop through it and return a single element for every object in the array. That way if you ever need to add items to the nav, its really easy you just create a new object. Or If alternatively we were in a situation where we wanted to make the nav data available through CMS data rather than hard coded, all we need to do is replace the data source, and we wouldnt need to make code changes to the JSX.